src/hooks/context/useResetContext.js

```diff
@@ -28,17 +28,14 @@ const useResetContext = () => {
   const setRefreshCount = useSetRecoilState(resetAtom);
   const refreshCount = useRecoilValue(resetAtom);
 
-  const {setProfileId} = useProfileHook();
+  const {resetAuth} = useUserAuthHook();
 
   const resetAllContext = () => {
     setRefreshCount(new Date().getTime());
     DiscoveryAction.resetAllDiscovery(discoveryDispatch);
     resetAllNews(newsDispatch);
     setMyProfileResetAll(profileDispatch);
-    setProfileId({
-      anonProfileId: null,
-      profileId: null
-    });
+    resetAuth();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `resetAllContext` function now calls `resetAuth` from `useUserAuthHook` instead of setting the profile ID to null using `setProfileId` from `useProfileHook`. This change seems to be in line with the overall shift from `useProfileHook` to `useUserAuthHook`. However, ensure that `resetAuth` properly resets the authentication state as expected.
```diff
-  const {setProfileId} = useProfileHook();
+  const {resetAuth} = useUserAuthHook();

   const resetAllContext = () => {
     setRefreshCount(new Date().getTime());
     DiscoveryAction.resetAllDiscovery(discoveryDispatch);
     resetAllNews(newsDispatch);
     setMyProfileResetAll(profileDispatch);
-    setProfileId({
-      anonProfileId: null,
-      profileId: null
-    });
+    resetAuth();
   };
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->